src/runtime_src/core/edge/user/aie/aiereg.h

```diff
@@ -0,0 +1,367 @@
+      {"Event_Broadcast_Block_East_Clr", 0x00034084 },
+      {"Event_Broadcast_Block_East_Value", 0x00034088},
+      {"Trace_Control0", 0x000340D0},
+      {"Trace_Control1", 0x000340D4},
+      {"Trace_Status", 0x000340D8},
+      {"Trace_Event0", 0x000340E0},
+      {"Trace_Event1", 0x000340E4},
+      {"Timer_Trig_Event_Low_Value", 0x000340F0},
+      {"Timer_Trig_Event_High_Value", 0x000340F4},
+      {"Timer_Low", 0x000340F8},
+      {"Timer_High", 0x000340FC },
+      {"Event_Status0", 0x00034200 },
+      {"Event_Status1", 0x00034204 },
+      {"Event_Status2", 0x00034208 },
+      {"Event_Status3", 0x0003420C },
+      {"Combo_event_inputs", 0x00034400 },
+      {"Combo_event_control", 0x00034404 },
+      {"Event_Group_0_Enable", 0x00034500 },
+      {"Event_Group_PC_Enable", 0x00034504 },
+      {"Event_Group_Core_Stall_Enable", 0x00034508},
+      {"Event_Group_Core_Program_Flow_Enable", 0x0003450C},
+      {"Event_Group_Errors0_Enable", 0x00034510},
+      {"Event_Group_Errors1_Enable", 0x00034514},
+      {"Event_Group_Stream_Switch_Enable", 0x00034518 },
+      {"Event_Group_Broadcast_Enable", 0x0003451C },
+      {"Event_Group_User_Event_Enable", 0x00034520},
+      {"Tile_Control", 0x00036030},
+      {"Tile_Control_Packet_Handler_Status", 0x00036034},
+      {"Tile_Clock_Control", 0x00036040},
+      {"CSSD_Trigger", 0x00036044},
+      {"Spare_Reg", 0x00036050},
+      {"Stream_Switch_Master_Config_ME_Core0", 0x0003F000},
+      {"Stream_Switch_Master_Config_ME_Core1", 0x0003F004 },
+      {"Stream_Switch_Master_Config_DMA0", 0x0003F008},
+      {"Stream_Switch_Master_Config_DMA1", 0x0003F00C },
+      {"Stream_Switch_Master_Config_Tile_Ctrl", 0x0003F010},
+      {"Stream_Switch_Master_Config_FIFO0", 0x0003F014},
+      {"Stream_Switch_Master_Config_FIFO1", 0x0003F018},
+      {"Stream_Switch_Master_Config_South0", 0x0003F01C},
+      {"Stream_Switch_Master_Config_South1", 0x0003F020},
+      {"Stream_Switch_Master_Config_South2", 0x0003F024},
+      {"Stream_Switch_Master_Config_South3", 0x0003F028},
+      {"Stream_Switch_Master_Config_West0", 0x0003F02C},
+      {"Stream_Switch_Master_Config_West1", 0x0003F030},
+      {"Stream_Switch_Master_Config_West2", 0x0003F034},
+      {"Stream_Switch_Master_Config_West3", 0x0003F038},
+      {"Stream_Switch_Master_Config_North0", 0x0003F03C},
+      {"Stream_Switch_Master_Config_North1", 0x0003F040},
+      {"Stream_Switch_Master_Config_North2", 0x0003F044},
+      {"Stream_Switch_Master_Config_North3", 0x0003F048},
+      {"Stream_Switch_Master_Config_North4", 0x0003F04C},
+      {"Stream_Switch_Master_Config_North5", 0x0003F050},
+      {"Stream_Switch_Master_Config_East0", 0x0003F054},
+      {"Stream_Switch_Master_Config_East1", 0x0003F058},
+      {"Stream_Switch_Master_Config_East2", 0x0003F05C},
+      {"Stream_Switch_Master_Config_East3", 0x0003F060},
+      {"Stream_Switch_Slave_ME_Core0_Config", 0x0003F100},
+      {"Stream_Switch_Slave_ME_Core1_Config", 0x0003F104},
```

Comment:
Syntax is inconsistent....
If there is a numeric value (e.g., 0, 1, 2, 3, etc.) within the register name, there is "usually" an underscore prior to the value, as show with:
`Stream_Switch_Slave_DMA_0_Config`
but that is not the case for:
`Stream_Switch_Slave_ME_Core1_Config`

In addition, it looks like if the numeric value is at the end of the register name, there is no underscore, for example:
`Stream_Switch_Master_Config_East3`

Is this truly the desired behavior?

Reply:
same as above

Reply:
Again not good :(